test/establishments/establishment.test.js

```diff
@@ -970,6 +972,89 @@ describe ("establishment", async () => {
+                .set('Authorization', authToken)
+                .send({
+                    serviceUsers: [
+                        {
+                            service: 'Carers of adults',
+                            other: "my example other"
+                        }
+                    ]
+                })
+                .expect('Content-Type', /json/)
+                .expect(200);
+            expect(otherOnNonOtherPostResponse.body.serviceUsers[0].id).toEqual(21);
+        
+            changeHistory =  await apiEndpoint.get(`/establishment/${establishmentUid}/serviceUsers?history=property`)
+            .set('Authorization', authToken)
+            .expect('Content-Type', /json/)
+            .expect(200);
+
+            chai.expect(changeHistory.body.serviceUsers.currentValue).to.eql([{ id: 21,
+                group: 'Carers',
+                service: 'Carers of adults'}]);
+            
+            // Should update service users with other and should update on an 'other service'
+            const otherOnOtherPostResponse = await apiEndpoint.post(`/establishment/${establishmentUid}/serviceUsers`)
+                .set('Authorization', authToken)
+                .send({
+                    serviceUsers: [
+                        {
+                            service: 'Older people not in above categories',
+                            other: "my example other"
+                        }
+                    ]
+                })
+                .expect('Content-Type', /json/)
+                .expect(200);
+            expect(otherOnOtherPostResponse.body.serviceUsers[0].id).toEqual(9);
+        
+            changeHistory =  await apiEndpoint.get(`/establishment/${establishmentUid}/serviceUsers?history=property`)
+            .set('Authorization', authToken)
+            .expect('Content-Type', /json/)
+            .expect(200);            
+
+            chai.expect(changeHistory.body.serviceUsers.currentValue).to.eql([{id :9, group :"Older people", service :"Older people not in above categories", other :"my example other"}]);
+
+            // Test with no other passed in on a 'other service'
+            const noOtherOnOtherPostResponse = await apiEndpoint.post(`/establishment/${establishmentUid}/serviceUsers`)
+                .set('Authorization', authToken)
+                .send({
+                    serviceUsers: [
+                        {
+                            service: 'Older people not in above categories'
+                        }
+                    ]
+                })
+                .expect('Content-Type', /json/)
+                .expect(200);
+            expect(noOtherOnOtherPostResponse.body.serviceUsers[0].id).toEqual(9);
+        
+            changeHistory =  await apiEndpoint.get(`/establishment/${establishmentUid}/serviceUsers?history=property`)
+            .set('Authorization', authToken)
+            .expect('Content-Type', /json/)
```

Comment:
Expecting on change history here is good too - namely, that when removing the optional other to the same service it is recognised as a change.

But you do need to expect on the change, not justr recall it.